### PR TITLE
Tab Directives 

### PR DESCRIPTION
**Pull request by jasperboyd**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/4628)

### CODE REVIEW

File: org/Hibachi/client/src/core/services/scopeservice.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {BaseService} from "./baseservice";
+
+class ScopeService { 
+
+    constructor(){
+
+    }
+
+    public locateParentScope = (scope, targetName) =>{
```
Comment (rmarchandTen24) [inline org/Hibachi/client/src/core/services/scopeservice.ts]: can we change the name to getRootParentScope

Comment (jasperboyd) [inline org/Hibachi/client/src/core/services/scopeservice.ts]: maybe getParentScope? it's not necessarily the root

Comment (rmarchandTen24) [inline org/Hibachi/client/src/core/services/scopeservice.ts]: the root is the top most parent and it looks like the while loop keeps going until it hits the top most parent. GetParent would imply moving up one space in the chain and would not apply here.

Comment (jasperboyd) [inline org/Hibachi/client/src/core/services/scopeservice.ts]: I see what you're thinking, root was just making me think of rootScope

Comment (rmarchandTen24) [inline org/Hibachi/client/src/core/services/scopeservice.ts]: getTopParentScope?

**Pull request merged**